Readme_Filtering.md

```diff
@@ -8,7 +8,7 @@
 Why do we need a filter functionality for SBOMs?  
 Well, not all SBOMs are really perfect, i.e. they may not contain
 what we expected - sometimes there are too many components, sometimes
-there are too few components.
+there are to few components.
```

Comment:
I think "too few" was correct here.

Reply:
ups, yes you are right @gernot-h . I corrected it. Thank you for reviewing